cmd/gokr-packer/packer.go

```diff
@@ -423,17 +457,48 @@ func logic() error {
 
 	partuuid := derivePartUUID(*hostname)
 	usePartuuid := true
+	var (
+		updateHttpClient         *http.Client
+		foundMatchingCertificate bool
+		updateBaseUrl            *url.URL
+	)
+
 	if *update != "" {
+		updateBaseUrl, err = url.Parse(*update)
+		if err != nil {
+			return err
+		}
+
+		updateHttpClient, foundMatchingCertificate, err = httpclient.GetTLSHttpClientByTLSFlag(useTLS, updateBaseUrl)
```

Comment:
I think we should skip this whole code branch when updateBaseUrl.Scheme is http (not https). That way, users can explicitly opt out of anything TLS-related by using a http:// update URL explicitly.

Reply:
They wouldn't opt-out, if the target instance still redirects to https. Instead they would get an certificate verification error if the CA is not in the system trust store. And what shall we do, if the tls flag is still present but http is used in the URL? This might be counterintuitive.

Reply:
> And what shall we do, if the tls flag is still present but http is used in the URL?

We could print an error in that case